package.json

```
@@ -83,6 +83,6 @@
     "preversion": "npm run build && (git diff --quiet || git commit -am build)"
   },
   "engines": {
-    "node": ">= 8.16.2"
+    "node": "^10 || ^12 || >=14"
```

```suggestion
    "node": ">=10"
```

Let's simplify this semver range.

We shouldn't explicitly block 11 and 13 unless there are known bugs.